package.json

```diff
@@ -14,6 +14,7 @@
   "scripts": {
     "build": "lage build",
     "build-tools": "lage build-tools",
+    "build-tokens": "transform-tokens --in ./packages/theming/theme-tokens/src/pipeline-input/token-input.json --out ./packages/theming/theme-tokens/src --p reactnative",
```

Comment:
That's fair, Travis said something similar. I can just have it output to reactnative directly.

Reply:
Do we need a folder named `react-native` at all? given that everything in this library will be react native? Could we just do something like `output` ?

Reply:
Ah that's the folder name that gets generated by the pipeline - since we use the reactnative platform, we get outputs in a reactnative folder (as opposed to web, or ios). I can see if I can add an option to change the output file name? But at the moment it's just how the pipeline works.

Reply:
By default the pipeline spits out everything, and `--platform` lets you filter to get just one or a few outputs. I could do something where if you only choose a single platform, it doesn't group the outputs into one folder per platform.

Reply:
The pipeline spitting out platform folders makes sense! Thanks for the info!

Reply:
I tried moving the output outside of src and then building broke. I'd like to check in what's here and figure that out in its own change.